### PR TITLE
HFR-SYNDI SUBTYPES AND SHIT ADDED

### PR DESCRIPTION
**Pull request by warface1234455**
Here is all the subtypes, simply just put the icon_states for the boxes, and the dmi files

### CODE REVIEW

File: code/game/objects/items/circuitboards/machine_circuitboards.dm
Comment (Addust) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm:1]: fix your shit

File: code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm [hunk 1]
```diff
@@ -15,6 +15,12 @@
 		if(. == FALSE)
 			break
 
+		if(syndicate_machine && !object.syndicate_machine)
+			. = FALSE
+
+		if(!syndicate_machine && object.syndicate_machine)
```
Comment (Addust) [inline code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm:21]: what the fuck does this bit do

Comment (ShoziOffical) [reply code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm:21]: so it wont fucking connect the normal hfr and syndi hfr together

File: code/game/objects/items/circuitboards/machine_circuitboards.dm
Comment (ShoziOffical) [reply code/game/objects/items/circuitboards/machine_circuitboards.dm:1]: done

File: code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm:21]: dont worry i tested all of them

Comment (Addust) [reply code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm:21]: amazing, didnt even know that'd be needed

File: code/game/objects/items/circuitboards/machine_circuitboards.dm
Comment (Addust) [reply code/game/objects/items/circuitboards/machine_circuitboards.dm:1]: shit fixed

**Pull request merged**